tests/std/tests/GH_001103_countl_zero_correctness/test.cpp

```diff
@@ -42,27 +42,27 @@ int main() {
 
     assert(_Countr_zero_bsf(static_cast<unsigned char>(0x00)) == 8);
     assert(_Countr_zero_bsf(static_cast<unsigned char>(0x13)) == 0);
-    assert(_Countr_zero_bsf(static_cast<unsigned char>(0x82)) == 1);
+    assert(_Countr_zero_bsf(static_cast<unsigned char>(0x80)) == 7);
     assert(_Countr_zero_bsf(static_cast<unsigned char>(0xF8)) == 3);
 
     assert(_Countr_zero_bsf(static_cast<unsigned short>(0x0000)) == 16);
     assert(_Countr_zero_bsf(static_cast<unsigned short>(0x0013)) == 0);
-    assert(_Countr_zero_bsf(static_cast<unsigned short>(0x8002)) == 1);

```

Comment:
why remove these tests? Instead of just adding the additional tests for large suffixes

Reply:
I don't think it adds much coverage, as we have ` == 3` tests

Reply:
Discussed: we'll have another PR to fix tests to keep this PR mirrored as it is.